### PR TITLE
Added steps for presenatation scenaario's

### PR DESCRIPTION
**Pull request by rajukakumanu86**

### CODE REVIEW

File: src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <LRN>38VYQTYFU3T0KUTUM3</LRN>
         <declarationType>Pbg</declarationType>
         <additionalDeclarationType>D</additionalDeclarationType>
-        <security>8</security>
+        <security>1</security>
```
Comment (vajarova) [inline src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml:11]: Are we not testing scenarios where the security is 0?

**Review by vajarova — CHANGES_REQUESTED** [review]

File: src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml (hunk 1 above)
Comment (rajukakumanu86) [reply src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml:11]: i can create 1 new scenario, but as we dont have form previous sections not added that one

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/runner/RunnerForPresentationNotificationIE13.scala [hunk 2]
```diff
@@ -24,6 +24,6 @@ import org.junit.runner.RunWith
   features = Array("src/test/resources/features"),
   glue = Array("uk.gov.hmrc.test.ui.cucumber.stepdefs"),
   plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json"),
-  tags = "@presentationNotification"
+  tags = "@presentationNotificationIe13"
 )
-class RunnerForPresentationNotification {}
+class RunnerForPresentationNotificationIE13 {}
```
[the remark this answers is not in the document]
Comment (rajukakumanu86) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/runner/RunnerForPresentationNotificationIE13.scala:29]: made it to only 1 runner

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/runner/RunnerForPresentationNotificationIE15.scala [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/*
```
[the remark this answers is not in the document]
Comment (rajukakumanu86) [reply src/test/scala/uk/gov/hmrc/test/ui/cucumber/runner/RunnerForPresentationNotificationIE15.scala:1]: changed it to 1 runner

File: src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml (hunk 1 above)
Comment (rajukakumanu86) [reply src/test/resources/xml/IE015SimplifiedPrelodgedDepartureDeclarationFull.xml]: added this pls check

**Review by vajarova — APPROVED** [review]

**Pull request merged**